mlir/lib/Dialect/QCO/IR/QCOOps.cpp

```
@@ -91,90 +92,132 @@ parseTargetAliasing(OpAsmParser& parser, Region& region,
+  p << " else";
+  printQubitsBlock(getElseRegion(), getQubits());
+  p.printRegion(getElseRegion(), /*printEntryBlockArgs=*/false);
+
+  p.printOptionalAttrDict((*this)->getAttrs());
+}
 
-  if (thenCount != elseCount) {
-    return parser.emitError(
-        parser.getCurrentLocation(),
-        "then/else qubit aliasing lists must be the same length");
+static void printTargetAliasing(OpAsmPrinter& printer, Operation* /*op*/,
```

Cppcheck (2.20.0)

[style] The function 'printTargetAliasing' is never used.

(unusedFunction)